build.sbt

```diff
@@ -0,0 +1,18 @@
+name := "Scala.js DOM"
+
+normalizedName := "scalajs-java-time"
+
+version := "0.0.1-SNAPSHOT"
+
+organization := "org.scala-js"
+
+scalaVersion := "2.11.7"
+
+scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings")
+
+homepage := Some(url("http://scala-js.org/"))
+
+licenses += ("MIT", url("http://opensource.org/licenses/mit-license.php"))
+
+lazy val root = project.in(file(".")).
+  enablePlugins(ScalaJSPlugin)
```

Comment:
Just `enablePlugins(ScalaJSPlugin)` at the top of the file. No need for a `lazy val root`